components/desktop/librecad/Makefile

```diff
@@ -0,0 +1,114 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Carsten Grzemba cgrzemba�@opencsw.org
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME =	LibreCAD
+COMPONENT_VERSION =	2.1.3
+COMPONENT_PROJECT_URL =	https://github.com/LibreCAD
+COMPONENT_FMRI =		desktop/librecad
+COMPONENT_SRC =		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+# COMPONENT_ARCHIV E=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH =	\
+# COMPONENT_ARCHIVE_URL=  https://github.com/$(COMPONENT_NAME)/releases/download/v$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_LICENSE =	Apache2-or-GPLv3
+COMPONENT_LICENSE_FILE = LICENSE
+# COMPONENT_LICENSE_FILE += LICENSE_Apache2.txt
+# COMPONENT_LICENSE_FILE += LICENSE_GPLv3.txt
+# COMPONENT_LICENSE_FILE += LICENSE_KST32B_v2.txt
+# COMPONENT_LICENSE_FILE += license-lc_opengost-fonts.txt
+# COMPONENT_LICENSE_FILE += LICENSE-MIT.txt
+# COMPONENT_LICENSE_FILE += gpl-2.0.txt
+# COMPONENT_LICENSE_FILE += gpl-2.0greater.txt
+
+COMPONENT_SUMMARY =	LibreCAD is a 2D CAD drawing tool based on the community edition of QCAD.
+COMPONENT_CLASSIFICATION = Applications/Graphics and Imaging
+
+COMPONENT_REVISION = $(shell cd $(COMPONENT_SRC); git rev-list HEAD --count)
+IPS_COMPONENT_VERSION = $(COMPONENT_VERSION)
+
+$(SOURCE_DIR)/.downloaded: $(ARCHIVES:%=$(USERLAND_ARCHIVES)%)
+	@echo "target .download"
+	[ -d $(SOURCE_DIR) ] || \
+	$(GIT) clone -b $(GIT_TAG) --single-branch --depth 1 $(GIT_REPO) $(SOURCE_DIR) && \
+	cd $(SOURCE_DIR) && $(GIT) log -1 --format=%H > .downloaded
+
+include ../../../make-rules/prep.mk
+# include ../../../make-rules/cmake.mk
+include ../../../make-rules/justmake.mk
+
+# publish: update-publish
+include ../../../make-rules/ips.mk
+
+GIT = git
+GIT_REPO = https://github.com/LibreCAD/LibreCAD.git
+GIT_TAG = $(COMPONENT_VERSION)
+GIT_BRANCH = master
+
+tolower = $(shell echo $(1) | tr '[A-Z]' '[a-z]')
```

Comment:
Below snipplet looks too ugly... Perhaps, better to patch librecad's Makefile to do correct thing on gmake install?